views/editChoral.pug

```diff
@@ -7,33 +8,24 @@ block content
       b
         i.fa.fa-dashboard
         |  Edit Choral
-  form.w3-container(method='POST' action='/chorals/edit/'+choralEdit.choralId id='mainForm').new-choral-form
+  form.w3-container(method='POST' action='/chorals/edit/'+ choralEdit.choralId id='mainForm').new-choral-form
```

Comment:
Could you add another form group, if you think it is a good idea:
.form-group
  label(for='choralIdInput') ID
  input#choralIdInput.form-control(name='id' type='text' readonly value=choralEdit.choralId)